lib/python/test/KHI_growthRate/MainTest.py

```diff
@@ -0,0 +1,38 @@
+import sys
```

Comment:
Please add the license header to all of your python script https://github.com/ComputationalRadiationPhysics/picongpu/blob/d9ab3532a2b0d10e752b5b701ec14254f486d176/lib/python/picongpu/plugins/plot_mpl/base_visualizer.py#L1-L7

Do not forget to set the year to `2022` only and add yourself as the author.